primary/src/core.rs

```diff
@@ -131,6 +139,13 @@ impl<PublicKey: VerifyingKey> Core<PublicKey> {
 
     #[instrument(level = "debug", skip_all)]
     async fn process_own_header(&mut self, header: Header<PublicKey>) -> DagResult<()> {
+        if header.epoch < self.committee.epoch() {
+            return Ok(());
```

Comment:
This can happen a lot because of asynchrony, we may have a debug but it is expected behaviour

Reply:
It's alright if it happens a lot - I am coming more from a place where I would like to have insight to confirm that this is actually happening. For ex if all the sudden in production our node is stop processing our own headers and this is the reason (without being able to recover) this could be an extra info to diagnose.

Reply:
got it, adding a debug